README.md

```diff
@@ -43,6 +54,8 @@ Test your installation using the following command (this will return also a list
 ``` bash
 python -m myosuite.tests.test_myo
 ```
+ 
+<!-- TODO : include visualize images? -->
```

Comment:
@raku-slyu @Vittorio-Caggiano -- Seems like some missing docs todo

Reply:
This was referring to the visualise env in the next line but not sure we need that still

Reply:
Remove the Todo in that case